azext_edge/edge/params.py

```diff
@@ -1027,3 +1028,63 @@ def load_iotops_arguments(self, _):
             help="File path containing or inline content for the version.",
             arg_group=None,
         )
+
+    with self.argument_context("iot ops clone") as context:
+        context.argument(
+            "summary_mode",
+            options_list=["--summary"],
+            arg_type=get_enum_type(SummaryMode, default=SummaryMode.SIMPLE.value),
+            help="Deployment summary option.",
+        )
+        context.argument(
+            "instance_name",
+            options_list=["--name", "-n"],
+            help="The model instance to clone.",
+        )
+        context.argument(
+            "resource_group_name",
+            options_list=["--resource-group", "-g"],
+            help="The resource group the model instance to clone resides in.",
+        )
+        context.argument(
+            "to_dir",
+            options_list=["--to-dir"],
+            help="The local directory the instance clone definitions will be stored in.",
+            arg_group="Local Target",
+        )
+        context.argument(
+            "template_mode",
+            options_list=["--mode"],
+            arg_type=get_enum_type(TemplateMode, default=TemplateMode.NESTED.value),
+            help="When mode 'nested' is used sub-deployments will be self-contained in the root deployment. "
```

Comment:
nit? When mode 'nested' is used,(comma) sub-deployments will be self-contained in the root deployment.